src/Common/src/Interop/Windows/advapi32/Interop.GetTokenInformation.cs

```diff
@@ -25,5 +25,13 @@ internal static partial class Advapi32
             SafeLocalAllocHandle TokenInformation,
             uint TokenInformationLength,
             out uint ReturnLength);
+
+        [DllImport(Interop.Libraries.Advapi32, CharSet = CharSet.Unicode, SetLastError = true)]
+        internal static extern bool GetTokenInformation(
+            IntPtr TokenHandle,
```

Comment:
Any reason you cannot use one of the overloads with the SafeLocalAllocHandle instead?

Reply:
I didn't want to refactor the code, since I wasn't sure which of the overrides would be appropriate.